packages/components/src/components/domainproperties/dataset/DatasetDesignerPanels.tsx

```diff
@@ -208,12 +260,17 @@ export class DatasetDesignerPanelImpl extends React.PureComponent<Props & Inject
     handleFileImport() {
         const { setSubmitting } = this.props;
         const { fileImportData, model } = this.state;
+        const parameters = {
+            name: model.name,
```

Comment:
we are already passing the `model.name` to the importData call as the queryName param, was there a specific reason that it needed to be passed as another param?

Reply:
followed the list designer approach here and introduced the 'name' property as opposed to 'queryName' that would look odd in ImportDatasetForm  formbean object in the controller side for the case when we need to get the dataset definition by name.